compiler-rt/test/profile/lit.cfg.py

```diff
@@ -147,6 +147,9 @@ def exclude_unsupported_files_for_aix(dirname):
 config.substitutions.append(
     ("%clang_pgouse=", build_invocation(clang_cflags) + " -fprofile-use=")
 )
+config.substitutions.append(
+    ("%clangxx_pgouse=", build_invocation(clang_cxxflags) + " -fprofile-use=")
```

Comment:
Is this used?

Reply:
`rg clangxx_pgouse compiler-rt/test/` => no result?

Reply:
oh you are correct.

I once used `clangxx_pgouse` in the new test for profile use, but later thought `clang -c` (run up to assembler phase to generate bitcode) together with `opt` and `llvm-lto` is better.